### PR TITLE
Developer sanjeev

### PR DESCRIPTION
**Pull request by sanjeevkumar0412**
Sentimental Prohibited keyword function added and CORS enable in flask

### CODE REVIEW

File: app/database_query_utils.py [hunk 1]
```diff
@@ -51,27 +52,27 @@ def get_all_record(self, server_name, database_name, client_id,table_name):
                 cursor.execute(raw_sql)
                 result = self.list_of_dictionary_conversion(cursor)
                 if len(result) > 0:
-                    api_object = self.global_utility.get_json_format(result)
-                    return api_object
+                    api_object = self.global_utility.get_json_format(result,SUCCESS)
+                    return api_object,SUCCESS
                 else:
-                    api_object = self.global_utility.get_json_format(result,400,True,'There is no record in the database')
```
Comment (solenkisudhiragreeya) [inline app/database_query_utils.py:57]: take the 400 code from error_code enums

File: app/model/sentiment_analysis.py [hunk 2]
```diff
@@ -142,13 +144,18 @@ def get_data_from_transcribe_table(self, server_name, database_name, client_id,a
...
+            trans_dic={}
             check_audio_id_exits = session.query(AudioTranscribe).filter(
                 AudioTranscribe.AudioFileName == audio_file).all()
+
             if len(check_audio_id_exits) > 0:
                 audio_id_query = session.query(AudioTranscribe.Id).filter(
                     AudioTranscribe.AudioFileName == audio_file)
                 query_audio_id_results = audio_id_query.all()
-                if len(query_audio_id_results) > 0:
+                check_chunk_exist = session.query(AudioTranscribeTracker.ChunkText).filter(
```
Comment (solenkisudhiragreeya) [inline app/model/sentiment_analysis.py:155]: validate this condition again

File: app/model/sentiment_analysis.py [hunk 3]
```diff
@@ -180,78 +193,129 @@ def get_data_from_transcribe_table(self, server_name, database_name, client_id,a
...
-            session.close()
+            try:
+                if len(check_audio_file_exits) > 0:
+                    sentiment_dic={}
+                    data = session.query(SentimentAnalysis).filter_by(AudioFileName=audio_file).all()
+                    sentiment_dic.update({"Id":data[0].Id,"ClientId":data[0].ClientId,
+                                          "AnalysisDateTime":data[0].AnalysisDateTime,"AudioFileName":data[0].AudioFileName,
+                                          "Created":data[0].Created,"SummaryReport":data[0].Summary,"Topics":data[0].Topics,
+                                          "FoulLanguage":data[0].FoulLanguage,"ActionItems":data[0].ActionItems,
+                                          "Owners":data[0].Owners,"SentimentScore":data[0].SentimentScore,
```
Comment (solenkisudhiragreeya) [inline app/model/sentiment_analysis.py:263]: suppose some value is not return by API then this code will be break. Apply the undefine and null check here

File: app/model/sentiment_analysis.py [hunk 4]
```diff
@@ -180,78 +193,129 @@ def get_data_from_transcribe_table(self, server_name, database_name, client_id,a
...
+                    AudioTranscribe.AudioFileName == audio_file).all()
+                if len(check_audio_file_exits) > 0:
+                    audio_id_query = session.query(AudioTranscribe.Id).filter(
+                        AudioTranscribe.AudioFileName == audio_file)
+                    query_audio_id_results = audio_id_query.all()
+                    check_chunk_exist = session.query(AudioTranscribeTracker.ChunkText).filter(
+                        AudioTranscribeTracker.AudioId == query_audio_id_results[0][0])
+                    # blank_emails = session.query(AudioTranscribeTracker).filter(AudioTranscribeTracker.ChunkText == '').all()
+                    chunk_results_check = check_chunk_exist.all()
+                    if len(query_audio_id_results) > 0 and chunk_results_check[0][0] !=None:
```
[the remark this answers is not in the document]
Comment (sanjeevkumar0412) [reply app/model/sentiment_analysis.py:211]: I have already handled this validation on the main function of sentiment.

**Comment by sanjeevkumar0412** [comment]
I have already handled this validation on the main function of sentiment.

**Pull request merged**